python/cuml/linear_model/logistic_regression.pyx

```diff
@@ -423,36 +475,44 @@ class LogisticRegression(Base):
         return accuracy_score(y, self.predict(X), handle=self.handle)
 
     def get_param_names(self):
-        return ["C", "penalty", "tol", "fit_intercept", "max_iter",
-                "linesearch_max_iter", "l1_ratio", "solver"]
+        return [
+            "C",
+            "penalty",
+            "tol",
+            "fit_intercept",
+            "max_iter",
+            "linesearch_max_iter",
+            "l1_ratio",
+            "solver",
+        ]
 
     def __getstate__(self):
         state = self.__dict__.copy()
         # Remove the unpicklable handle.
-        if 'handle' in state:
-            del state['handle']
+        if "handle" in state:
+            del state["handle"]
```

Comment:
No need to delete the cuML handle anymore, it has been pickleable for a while (I know this wasn't intro'd in the PR, but might be a good chance to remove old code :) )

Reply:
Will do 👍 . Always good to remove old code :)

Reply:
Resolved by https://github.com/rapidsai/cuml/pull/2403/commits/e824ef4c2d8da3b017ec59b8ef4bbe2006c2e91c